examples/exposure_fixed.py

```diff
@@ -10,6 +10,6 @@
 
 preview_config = picam2.preview_configuration()
 picam2.configure(preview_config)
-
-picam2.start({"ExposureTime": 10000, "AnalogueGain": 1.0})
+picam2.set_controls({"ExposureTime": 10000, "AnalogueGain": 1.0})
+picam2.start()
```

Comment:
It looks to me like fixing up the examples here leaves quite a string of commits where the test set is broken, is that right? I'm wondering if we should really squash this with the commit that changed the start function signature.

Though I'm still unsure about the change to the start function. I'm quite keen that all the state associated with a configuration is "obvious", and there isn't anything hidden in the Picamera2 object. The idea being that if you call start the with the same configuration and parameters, you're guaranteed the same outcome.

Reply:
I think I'm probably OK with using the set_controls function instead of passing controls in start, I'm just struggling a bit to get used to the idea. I think I remain nervous about separating stream configuration and controls any more, because I think there are particular use cases where you want to specify appropriate defaults both for the configuration and for specific controls.